src/boolean3.cpp

```diff
@@ -35,7 +35,7 @@ vec2 Interpolate(vec3 pL, vec3 pR, double x) {
   const bool useL = fabs(dxL) < fabs(dxR);
   const vec3 dLR = pR - pL;
   const double lambda = (useL ? dxL : dxR) / dLR.x;
-  if (!isfinite(lambda) || !isfinite(dLR.y) || !isfinite(dLR.z))
+  if (!std::isfinite(lambda) || !std::isfinite(dLR.y) || !std::isfinite(dLR.z))
```

Comment:
Looks good, but for my own info, what is the difference between e.g. `isfinite` and `std::isfinite`?

Reply:
One is defined in cmath without namespace, one is in C++. I think they are practically the same.